docs/module-sources.rst

```diff
@@ -26,7 +26,13 @@ There are a few properties common to all source types ``only-arches``,
 
 ``only-arches`` and ``skip-arches`` are used to build the source only
 on the specified architecture or skip building it on the specified
-architecture respectively. This is often useful for binary packages:
+architecture respectively. This is often useful for binary packages.
+
+.. note::
+
+  Note that ``only-arches`` or ``skip-arches`` in a source, restricts
+  only `that` source to the specified architecture(s) and nothing else.
+  The rest, if not restricted, is built for all available architectures.
```

Comment:
Wouldn't just saying this be clear:

> ``only-arches`` and ``skip-arches`` are used to `download` the source only on the specified architecture.

This section is about sources so shouldn't mention building at all?

Reply:
How about now?